lib/screens_web/router.ex

```diff
@@ -13,7 +13,7 @@ defmodule ScreensWeb.Router do
   end
 
   pipeline :redirect_prod_http do
-    if Application.get_env(:screens, :redirect_http?) do
+    if Application.compile_env(:screens, :redirect_http?) do
```

Comment:
Oh interesting, I didn't know that these `pipeline` blocks were evaluated at compile time. The compiler complained about this?

Reply:
Yes, this was the one spot in our code that it warned about this.